### PR TITLE
Added ModelConverter

### PR DESCRIPTION
**Pull request by Knuxfan24**
Seems like something that should here.

### CODE REVIEW

File: docs/tools/hedgehog-engine/common/models.md [hunk 1]
```diff
@@ -6,4 +6,11 @@ description: Model tools for the Hedgehog Engine
...
 
-[GitHub Repository :fontawesome-brands-github:](https://github.com/DarioSamo/libgens-sonicglvl){ .md-button .md-button--primary target="_blank"}
\ No newline at end of file
+[GitHub Repository :fontawesome-brands-github:](https://github.com/DarioSamo/libgens-sonicglvl){ .md-button .md-button--primary target="_blank"}
+
+## ModelConverter
+
+Converts Assimp supported model formats into a Hedgehog Engine model file, with various batch scripts for the different games.
+
+[Download :fontawesome-solid-download:](https://github.com/blueskythlikesclouds/ModelConverter/releases){ .md-button .md-button--primary}
```
Comment (RagdollClash) [inline docs/tools/hedgehog-engine/common/models.md]:
```suggestion
[Download :fontawesome-solid-download:](https://github.com/blueskythlikesclouds/ModelConverter/releases/latest){ .md-button .md-button--primary}

```

**Review by RagdollClash — CHANGES_REQUESTED** [review]
Use `/latest` to direct people to the latest release. Otherwise, LGTM.

**Comment by Knuxfan24** [comment]
The review also made me notice that a few errors had snuck in on the General Audio page too (which is what I copied for reference, hence the lack of /latest in the releases URL).

**Review by RagdollClash — APPROVED** [review]
LGTM

**Pull request merged**